File: TypeScript/Card.ts
export class Card {

    name: string;
    wrongLast: boolean;

    constructor(name: string, wrongLast: boolean) {
        this.name = name;
        this.wrongLast = wrongLast;
    }

    wasWrongLast(): boolean {
        return this.wrongLast;
    }
}

File: Java/Card.java
public class Card {

    private String name;
    private boolean wrongLast;

    public Card(String name, boolean wrongLast) {
        this.name = name;
        this.wrongLast = wrongLast;
    }

    public boolean wasWrongLast() {
        return wrongLast;
    }

    public String getName() {
        return name;
    }
}
